### PR TITLE
fix: bulk-edit-ui-fixes

### PR DESCRIPTION
**Pull request by rish2320**
# Description
Multiple Bulk Edit UI fixes.

Fixes # [3511](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=3511)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested all the cases manually.
# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/bulkEdits/bulkEdit.scss [hunk 1]
```diff
@@ -66,19 +66,21 @@
...
     background: white;
+    .react-monaco-editor-container {
+        min-height: calc(60vh - 97px);
+    }
 }
 
 .bulk-output-body {
     letter-spacing: 0.2px;
-    height: calc(100vh - 600px);
+    height: calc(40vh - 36px);
```
Comment (milindk8) [inline src/components/bulkEdits/bulkEdit.scss:83]: have you tested by changing screen heights.

File: src/components/common/header/PageHeader.tsx [hunk 2]
```diff
@@ -159,7 +160,7 @@ function PageHeader({
                     </span>
                     {additionalHeaderInfo && additionalHeaderInfo()}
                     {isBreadcrumbs && breadCrumbs()}
-                    {isTippyShown && (
+                    {isTippyShown && headerName != "Bulk Edit" && (
```
Comment (milindk8) [inline src/components/common/header/PageHeader.tsx:163]: create constant for bulk edit and use it here and at the bottom

File: src/components/bulkEdits/bulkEdit.scss (hunk 1 above)
Comment (rish2320) [reply src/components/bulkEdits/bulkEdit.scss:83]: yes

File: src/components/common/header/PageHeader.tsx (hunk 2 above)
Comment (rish2320) [reply src/components/common/header/PageHeader.tsx:164]: done

**Review by milindk8 — APPROVED** [review]

**Review by milindk8 — APPROVED** [review]

File: src/components/bulkEdits/BulkEdits.tsx [hunk 3]
```diff
@@ -772,12 +744,12 @@ export default class BulkEdits extends Component<BulkEditsProps, BulkEditsState>
...
 
         return (
-            <div className="fs-13">
+            <div className="fs-13" >
                 <PageHeader
                     headerName="Bulk Edit"
                     isTippyShown={true}
                     TippyIcon={Question}
-                    tippyMessage={'Learn more'}
+                    tippyMessage={"Run scripts to bulk edit configurations for multiple devtron components."}
```
Comment (vivek-devtron) [inline src/components/bulkEdits/BulkEdits.tsx:752]: It can be written as string

File: src/components/common/header/PageHeader.tsx [hunk 4]
```diff
@@ -177,6 +179,23 @@ function PageHeader({
...
+                            className="w-300 h-100 fcv-5"
+                            placement="bottom"
+                            Icon={QuestionFilled}
+                            heading={headerName}
+                            infoText={tippyMessage}
+                            showCloseButton={true}
+                            trigger="click"
+                            interactive={true}
+                            documentationLink={tippyRedirectLink}
+                            documentationLinkText={"Learn More"}
```
Comment (vivek-devtron) [inline src/components/common/header/PageHeader.tsx:194]: it can be written as documentationLinkText="Learn More"

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**